src/mono/netcore/nuget/Microsoft.NET.Runtime.MonoAOTCompiler.Task/Sdk/Sdk.props

```diff
@@ -0,0 +1,20 @@
+<!--
```

Comment:
IIUC, this package is not meant to be used as a SDK. So, this file should instead be named `build/Microsoft.NET.Runtime.MonoAOTCompiler.Task.props`.

Reply:
Removing the file as the NuPkg will no longer import the UsingTask.

Reply:
It would still be useful to have `build/Microsoft.NET.Runtime.MonoAOTCompiler.Task.props` that populates a `$(MonoAOTCompilerTaskAssemblyPath)`, so that the project using this nuget can just use that, instead of having to construct the path. msbuild will automatically import this file.

Reply:
Something like
```
<PropertyGroup>
  <MonoAOTCompilerTaskAssemblyPath>tools\net5.0\MonoAOTCompiler.dll</MonoAOTCompilerTaskAssemblyPath>
</PropertyGroup>
```

Reply:
`<MonoAOTCompilerTaskAssemblyPath>$(MSBuildThisFileDirectory)..\tools\net5.0\MonoAOTCompiler.dll</MonoAOTCompilerTaskAssemblyPath>`

Reply:
Thanks, done!